massa-network-exports/src/establisher.rs

```diff
@@ -102,7 +102,9 @@ mod types {
             // Number of connections to queue, set to the hardcoded value used by tokio
             socket.listen(1024)?;
 
-            Ok(DefaultListener(TcpListener::from_std(socket.into())?))
+            let socket: std::net::TcpListener = socket.into();
+            socket.set_nonblocking(true).unwrap();
```

Comment:
should propagate error

Reply:
done https://github.com/massalabs/massa/pull/3685/commits/210e52e368844ba0859b46d7f0224a035fed021f